src/ADAL.PCL.CoreCLR/PlatformInformation.cs

```diff
@@ -61,5 +61,20 @@ public override string GetDeviceModel()
             // Since ADAL .NET may be used on servers, for security reasons, we do not emit device type.
             return null;
         }
+
+        public override string GetApplicationName()
+        {
+            return base.ToString();
+        }
+
+        public override string GetApplicationVersion()
+        {
+            return base.ToString();
```

Comment:
if only base is being called then we should not really be overriding the method.